GitHub-Test-3/emily_Regex2_sonnets.md

```diff
@@ -0,0 +1,26 @@
+# Regex  Assignment 2
+## Emily Wargo
+The first thing I had to do to the sonnet  was seperate each individual line :
+```
+Find: .+
+Replace: <line>\0</line>
+```
+Next, I wanted to find all of the sonnet titles and put `<sonnet>` tags around them.I used the characters `[IVXLC]` so it could recognize the letters as roman numerals: 
```

Comment:
I agree with this.
This commenting is called Line-Commenting!

Reply:
Actually, I would recommend a change here, in the way you're grabbing Roman numerals.